datalake/src/main/resources/duties/7.2.17/aws/enterprise.json

```diff
@@ -23,7 +23,8 @@
       "nodeCount": 2,
       "type": "CORE",
       "recoveryMode": "MANUAL",
-      "recipeNames": []
+      "recipeNames": [],
+      "hints": ["az1","az2"]
```

Comment:
Where did these "az" names come from?

Reply:
So could it be an arbitrary name like "A", "B", "C" or any random text? Should these not be well-known constants or rather enum values?

Reply:
I agree with Denes. Maybe some pre-defined enums would be nice, like AZ1, AZ2, AZ3

Reply:
Can node count be more than number of hints? As per implementation, looks like nodecount <= number of hints. if its more than hints, exception will be thrown.